docs/release-promotion/mobile/howto-rc.md

```diff
@@ -60,34 +60,15 @@ python tools/buildfarm/release/trigger_action.py \
 unset PROMOTE_TASK_ID
 ```
 
-This will show you a task definition and ask if you want to submit it (y/n). If you're ready to ship, choose `y`. The ship action taskId will be near the bottom of the output; this taskId is also used as the task graph ID for the ship graph.
+  * The `taskId` of the action task will be the `taskGroupId` of the next graph.
 
-For example, the last output from `trigger_action.py` will look something like this:
```

Comment:
I agree it's simpler now to remove all this context and simply say the. `The `taskId` of the action task will be the `taskGroupId` of the next graph`

++